docs/user/troubleshooting-guides/03-00-cannot-connect-to-service/03-02-404-not-found.md

```diff
@@ -0,0 +1,49 @@
+# 404 Not Found
+
+## Symptom
+
+When you try to reach your Service, you get `404 Not Found` in response.
+
+## Remedy
+
+Make sure that the following conditions are met:
+
+- Proper Oathkeeper Rule has been created:
+
+  ```bash
+  kubectl get rules.oathkeeper.ory.sh -n {NAMESPACE}
+  ```
+
+  >**TIP:** Name of the Rule consists of the name of the APIRule and a random suffix.
+
+- Proper VirtualService has been created:
+
+  ```bash
+  kubectl get virtualservices.networking.istio.io -n {NAMESPACE}
+  ```
+
+  >**TIP:** Name of the VirtualService consists of the name of the APIRule and a random suffix.
+
+Sometimes Oathkeeper Maester controller stops reconciling Rules on long-living clusters. This behavior might result in random `404 Not Found` responses because Oathkeeper does not contain Rules reflecting the actual state of the cluster. A simple restart of the Pod resolves the problem. To verify if that it is the cause of the issue you encountered, follow these steps:
```

Comment:
```suggestion
Sometimes, the Oathkeeper Maester controller stops reconciling Access Rules on long-living clusters. This behavior might result in the random `404 Not Found` responses because Oathkeeper does not contain Access Rules reflecting the actual state of the cluster. A simple restart of the Pod resolves the problem. To verify if that it is the cause of the issue you encountered, follow these steps:
```